submissions/Jemikalajah-Philip/Html/homepage.html

```diff
@@ -39,7 +39,7 @@
                         <div class="navbar">
                             <a href="homepage.html">Home</a>
                             <a href="#">Help</a>
-                            <a href="#">Contact</a>
+                            <a href="../php/create.php">Seller</a>
```

Comment:
You changed the directory's name but you are not refactoring path to your `create.php`